services/sleeper/src/sleeper/main.py

```diff
@@ -0,0 +1,110 @@
+import os
+import json
+import random
+import time
+import subprocess
+
+
+def get_from_environ(key, default=None):
+    """Returns a value from the environ if not found None"""
+    return os.environ.get(key, default)
+
+
+def get_random_sleep():
+    """Returns a random amount of sleep between 1 and 9"""
+    return random.randint(1, 9)
+
+
+def cast_bool(value):
+    """Transformas a string into a boolean"""
+    return value.lower() in {"true", "yes", "1"}
+
+
+def validate_not_negative_int(number):
```

Comment:
name here is misleading when i read the doc

Reply:
you are correct, this has been renamed now.